Assets/Mirror/Editor/Weaver/Processors/MethodProcessor.cs

```diff
@@ -35,6 +35,11 @@ public static MethodDefinition SubstituteMethod(TypeDefinition td, MethodDefinit
             string newName = RpcPrefix + md.Name;
             MethodDefinition cmd = new MethodDefinition(newName, md.Attributes, md.ReturnType);
 
+            // force new usercode_cmd to be private.
+            // this prevents users from mistakenly calling weaver generated methods in dropdown menus (such as buttons)
```

Comment:
```suggestion
            // otherwise the generated User_Cmd could be assigned to UnityEvents in the Inspector
            // (User_Cmd() is only called by Invoke_Cmd in this class)
```